/**:
  ros__parameters:
    devices: ["*"]
    traffic_reader_port: 7636
    monitor_program: "greengrass"
    crc_error_check_duration: 1
    crc_error_count_threshold: 1
    ip_reasm_fails_check_duration: 1
    ip_reasm_fails_count_threshold: 1
